application/src/main/java/org/togetherjava/tjbot/commands/basic/SuggestionsUpDownVoter.java

```diff
@@ -56,7 +58,15 @@ private static void reactWith(@NotNull String emoteName, @NotNull String fallbac
                     "Unable to vote on a suggestion with the configured emote ('{}'), using fallback instead.",
                     emoteName);
             return message.addReaction(fallbackUnicodeEmote);
-        }).queue();
+        }).queue(ignored -> {
+        }, exception -> {
+            if (exception instanceof ErrorResponseException responseException
+                    && responseException.getErrorResponse() == ErrorResponse.REACTION_BLOCKED) {
```

Comment:
Maybe add a quick comment to explain this use case:
```java
// User blocked the bot, hence the bot can not add reactions to their messages. Nothing we can do here.
return;

Reply:
sure.